testsuite/src/it/java/io/quarkus/ts/startstop/utils/WhitelistLogLines.java

```diff
@@ -89,9 +106,28 @@
         switch (OS.current()) {
             case MAC:
                 return new Pattern[] {
+                		COMMON_WARNING_VERTX_CACHE_DIRECTORY,
+                		COMMON_WARNING_DUPLICATE_ENTRY_NATIVE,
+                		COMMON_WARNING_DUPLICATE_ENTRIES_NATIVE,
+                        Pattern.compile(".*objcopy executable not found in PATH. Debug symbols will not be separated from executable.*"),
```

Comment:
should be constants, shared on Win + Mac

Reply:
Done